dags/operational_monitoring.py

```diff
@@ -0,0 +1,31 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from utils.gcp import gke_command
+
+default_args = {
+    "owner": "msamuel@mozilla.com",
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "msamuel@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2021, 6, 3),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+PROJECT_ID="operational_monitoring"
+
+with DAG(PROJECT_ID, default_args=default_args, schedule_interval="@daily") as dag:
+    operational_monitoring = gke_command(
+        task_id="run_operational_monitoring",
+        command=[
+            "bqetl",
+            "opmon",
+            "run",
+            "--submission-date={{ ds }}",
+        ],
+        docker_image="mozilla/bigquery-etl:latest",
+        dag=dag,
+    )
```

Comment:
This should run after `telemetry_derived__main_nightly__v1` (https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_main_summary.py#L282). `telemetry_derived__main_nightly__v1` needs to be specified as an upstream dependency and the `schedule_interval` should be set to `0 3 * * *` (by then the upstream task should have finished)